packages/finicity-adapter/src/adapter.ts

```diff
@@ -225,8 +226,13 @@ export class FinicityAdapter implements WidgetAdapter {
       case "credentialsUpdated":
         connection.status = ConnectionStatus.CONNECTED;
         break;
+      case "done":
+        return connection;
```

Comment:
This change fixed a problem I noticed where sometimes it seemed like there was a race between events and if this event happened first then the member_guid in the postmessage was incorrect.